iron-list.html

```diff
@@ -809,6 +776,7 @@
       if (this.isAttached && !this._itemsRendered && this._isVisible && requiresUpdate) {
         this._lastPage = 0;
         this._update();
+        this._scrollHandler();

```

Comment:
@kevinpschaaf This was a bug I found. As we know, calling `_render` after handling splices could result in a different physical item heights. We call `_update` which then calls `_updateMetrics`. At this point every thing looks great, but we forgot to recycle. After updating the metrics, it might need to recycle if the entry point to `_update`  isn't  `_scrollHandler` itself.  I will add a comment, I know this is so easy to forget.